types/braintree/index.d.ts

```diff
@@ -0,0 +1,1749 @@
+// Type definitions for braintree 2.16
+// Project: https://github.com/braintree/braintree_node
+// Definitions by: Sam Rubin <https://github.com/smrubin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
```

Comment:
We're on 3.3 aren't we?

Reply:
Yes. This flag specifies that the TS definition supports 2.1 and above. The compiler ran fine through the definition, so I must not have used anything introduced in >3. This option was recommended by the linter, but happy to change if you think we should only support newer versions?

Reply:
Had to up this to 2.8 for never for the new ValidatedResponse.